src/it/java/org/jboss/tools/intellij/quarkus/utils/GlobalUtils.java

```diff
@@ -0,0 +1,180 @@
+/*******************************************************************************
+ * Copyright (c) 2020 Red Hat, Inc.
+ * Distributed under license by Red Hat, Inc. All rights reserved.
+ * This program is made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution,
+ * and is available at http://www.eclipse.org/legal/epl-v20.html
+ *
+ * Contributors:
+ * Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+package org.jboss.tools.intellij.quarkus.utils;
+
+import com.intellij.remoterobot.RemoteRobot;
+import com.intellij.remoterobot.fixtures.ComponentFixture;
+import com.intellij.remoterobot.utils.WaitForConditionTimeoutException;
+import org.apache.commons.io.FileUtils;
+import org.jboss.tools.intellij.quarkus.fixtures.dialogs.IdeFatalErrorsDialogFixture;
+import org.jboss.tools.intellij.quarkus.fixtures.dialogs.TipOfTheDayDialogFixture;
+import org.jboss.tools.intellij.quarkus.fixtures.dialogs.WelcomeFrameDialogFixture;
+import org.jboss.tools.intellij.quarkus.fixtures.mainIdeWindow.IdeStatusBarFixture;
+
+import java.io.File;
+import java.io.IOException;
+import java.time.Duration;
+
+import static com.intellij.remoterobot.search.locators.Locators.byXpath;
+import static com.intellij.remoterobot.stepsProcessing.StepWorkerKt.step;
+
+/**
+ * Static utilities that assist and simplify manipulation with the IDE and with the project
+ *
+ * @author zcervink@redhat.com
+ */
+public class GlobalUtils {
+
+    public static String projectPath = "";
+
+    public static void closeTheTipOfTheDayDialog(RemoteRobot remoteRobot) {
+        step("Close the 'Tip of the Day' Dialog", () -> {
+            final TipOfTheDayDialogFixture tipOfTheDayDialogFixture = remoteRobot.find(TipOfTheDayDialogFixture.class, Duration.ofSeconds(20));
+            tipOfTheDayDialogFixture.button("Close").click();
+        });
+    }
+
+    public static void waitUntilTheProjectImportIsComplete(RemoteRobot remoteRobot) {
+        step("Wait until the project import is complete", () -> {
+            while (true) {
```

Comment:
There is a built-in method for conditional waits in `com.intellij.remoterobot.utils.RepeatUtilsKt` called `waitFor`. Please use that instead of endless loops.

Reply:
@jrichter1 Thank you very much for this advice. The infinite while loops have been replace with the waitFor() method in the **19ede0b** commit.